nixos/modules/services/hardware/nbfc-linux.nix

```diff
@@ -0,0 +1,61 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+with lib;
```

Comment:
with lib; is a bad practice, inherit the functions into scope from lib, or call them on lib directly.

Reply:
removed - all lib functions were called on lib directly